docs/setup/docker.asciidoc

```diff
@@ -9,14 +9,15 @@ found on https://github.com/elastic/kibana-docker/tree/{branch}[GitHub].
 
 === Image types
 
-The images are available in two different configurations or "flavors". The
-`x-pack` flavor, which is the default, ships with X-Pack features
-pre-installed. The `oss` flavor does not include X-Pack, and contains only
-open source Kibana.
-
-NOTE: https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] is
-pre-installed in the default image. With X-Pack installed, Kibana expects to
-connect to an Elasticsearch cluster that is also running X-Pack.
+These images are free to use under the Elastic license. They contain open source, 
+free commercial, and paid commercial features.  
```

Comment:
I totally thought I added this comment before, but apparently not:

nit: Perhaps "and access to paid commercial features" would be more clear?